creator-node/src/index.js

```diff
@@ -39,7 +39,11 @@ const getMode = () => {
 const startApp = async () => {
   logger.info('Configuring service...')
 
-  await config.asyncConfig()
+  try {
```

Comment:
i merged this fix to master already, can revert this change (and rebase)

Reply:
done